<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QWidget" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>551</width>
     <height>482</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>16</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Pilih Destinasi Wisata:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cmb_destinasi">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Pilih Tanggal Kunjungan</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCalendarWidget" name="calendar">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Jumlah Pengunjung:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="sb_pengunjung"/>
    </item>
    <item>
     <widget class="QLabel" name="label_harga">
      <property name="text">
       <string>Total Harga: 0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_beli">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Beli Tiket</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
